pkg/providers/github.go

```diff
@@ -0,0 +1,34 @@
+package providers
```

Comment:
I don't see any tests for this.  I know it's challenging to test the implementation of a library, but we should at least test our logic that uses the library (setting up a fake `github` package and testing that our provider works, and that our code that drives the provider in `NewAuthorizer()` works as well.